content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

```diff
@@ -64,27 +64,20 @@ If you create a new clone of the repository, you won't lose any of your Git hist
 
    {% endtip %}
 
-8. Check the existing remote name for your repository. For example, `origin` or `upstream` are two common choices.
+8. Add a new remote name with the URL you copied in step 7 for your repository. For example, `origin` or `upstream` are two common choices.
```

Comment:
I would remove references to other steps, since this won't update automatically if other steps are added in the future.

```suggestion
8. Add a new remote name with the URL you copied for your repository. For example, `origin` or `upstream` are two common choices.
```